eigenda/config.go

```diff
@@ -44,6 +47,10 @@ type Config struct {
 	// The blob encoding version to use when writing blobs from the high level interface.
 	PutBlobEncodingVersion codecs.BlobEncodingVersion
 
+	// ETH vars
+	EthRPC         string
+	SvcManagerAddr string
```

Comment:
We should update the readme with a table of these addresses across Holesky and Mainnet, or even better link to the canonical EigenDA contract readme where those addresses are written down.